.github/CODEOWNERS

```
@@ -5,30 +5,42 @@
 
 # Tendermint2 (Gno version).
 tm2/*                        @gnolang/tech-staff
-tm2/pkg                      @jaekwon @moul
+tm2/pkg                      @jaekwon @piux2 @moul
 # TODO: add per package exceptions
 
-
 # Docs & Content.
 docs/                        @gnolang/devrels @gnolang/tech-staff
```

```suggestion
docs/                        @gnolang/devrels
```

So that at least one devrel checks all doc changes.

---

Love this